superdesk/text_checkers/ai/__init__.py

```diff
@@ -0,0 +1,82 @@
+# Copyright 2020 Sourcefabric z.u. and contributors.
+#
+# For the full copyright and license information, please see the
+# AUTHORS and LICENSE files distributed with this source code, or
+# at https://www.sourcefabric.org/superdesk/license
+
+from superdesk.resource import Resource
+from superdesk.services import BaseService
+from superdesk.errors import SuperdeskApiError
+from .. import tools
+from .base import registered_ai_services, AIServiceBase
+import superdesk
+
+AUTO_IMPORT = True
+
+
+class AIResource(Resource):
+    schema = {
+        'service': {
+            'type': 'string',
+            'required': True,
+        },
+        'item_id': {
+            'type': 'string',
+            'required': True,
+        },
+    }
+    datasource = {
+        'projection': {'analysis': 1}
```

Comment:
I think we don't have `analysis` defined in the schema

Reply:
`analysis` is used for returned value. If I don't put the project here, it will be removed by Eve.

Reply:
you can also switch off projection per domain see https://github.com/superdesk/superdesk-core/pull/1961/files
but having `'projection': {'analysis': 1}` is also 👍